gatsby-browser.js

```diff
@@ -25,3 +25,29 @@ export const onClientEntry = () => {
     window.location.pathname = `/en${window.location.pathname}`
   }
 }
+
+export const onRouteUpdate = ({location}) => {
+  anchorScroll(location);
+  return true;
+};
+
+export const shouldUpdateScroll = ({
+  routerProps: { location },
+  getSavedScrollPosition
+}) => {
+  if (location && location.hash) {
+    anchorScroll(location);
+    return false;
+  } else {
+    return true;
+  }
+}
+
+function anchorScroll(location){
+  if (location && location.hash) {
+    setTimeout(() => {
+      const elm = document.querySelector(`${location.hash}`);
+      elm.scrollIntoView({ behavior: "smooth" });
```

Comment:
Do we have any polyfill for this? Or we just support it for browsers that have this capability and are OK it jumps for the others.